graphql.go

```diff
@@ -122,6 +123,13 @@ func MaxParallelism(n int) SchemaOpt {
 	}
 }
 
+// MaxQueryLength specifies the maximum allowed query length in bytes. The default is 0 which disables max length checking.
+func MaxQueryLength(n int) SchemaOpt {
```

Comment:
Nit: `MaxQueryLengthBytes` might be more readable to the user to avoid having to check what is the unit.